services/database/vector/threats.yaml

```diff
@@ -0,0 +1,129 @@
+      - CCC.TH10 # Alerts are Intercepted
+      - CCC.TH11 # Event Notifications are Incorrectly Triggered
+      - CCC.TH12 # Resource Constraints are Exhausted
+      - CCC.TH13 # Resource Tags are Manipulated
+      - CCC.TH14 # Older Resource Versions are Exploited
+      - CCC.TH15 # Automated Enumeration and Reconnaissance by Non-human Entities
+      - CCC.TH16 # Logging and Monitoring are Disabled
+      - CCC.TH17 # Unauthorized Network Access via Misconfigured Rules
+threats:
+  - id: CCC.Vector.TH01
+    title: Embedding Extraction and Model Inversion
+    description: Attackers may infer or reconstruct original data by probing vector
+      similarity APIs, especially with unrestricted access. This enables model inversion
+      attacks, membership inference, and unauthorized data leakage from stored embeddings.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F01
+          - CCC.Vector.F03
+          - CCC.Vector.F06
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-SEC-002
+  - id: CCC.Vector.TH02
+    title: Embedding and Index Poisoning
+    description: Adversaries may insert malicious or adversarial vectors into the index
+      through ingestion endpoints, polluting the dataset and degrading search quality,
+      or subtly steering results toward specific outcomes.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F02
+          - CCC.Vector.F05
+          - CCC.Vector.F07
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-SEC-009
+  - id: CCC.Vector.TH03
+    title: Cross-modal or Metadata Leakage
+    description: Attackers may infer sensitive information through metadata filters
+      or by correlating embeddings across modalities (e.g., voice and face), bypassing
+      surface-level access controls.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F04
+          - CCC.Vector.F10
+    external-mappings:
+      - reference-id: FINOS-AIR
+        identifiers:
+          - AIR-RC-022
+  - id: CCC.Vector.TH04
+    title: Denial of Service via Query or Ingestion Flooding
+    description: High-volume ingestion or excessive low-latency similarity queries can
+      overwhelm vector indexing services, degrading performance for legitimate users
+      and potentially exhausting system resources.
+    capabilities:
+      - reference-id: CCC
+        identifiers:
+          - CCC.Vector.F05
+          - CCC.Vector.F06
+    external-mappings:
+      - reference-id: MITRE-ATT&CK
+        identifiers:
+          - T1499 # Endpoint Denial of Service
+          - T1498 # Network Denial of Service
+          - T1496 # Resource Hijacking
+  - id: CCC.Vector.TH05
+    title: Index Corruption or Downgrade
+    description: Attackers with unauthorized access or excessive permissions may tamper
```

Comment:
This one sounds like a specific instance of `CCC.Core.TH01 Access Control is Misconfigured`